Tools/machines/crusher-olcf/crusher_warpx.profile.example

```diff
@@ -2,6 +2,7 @@
 #export proj=<yourProject>
 
 # required dependencies
+module load cce
```

Comment:
The default version, which loads `13.0.0`, has the same linker issue.

Specifically this version is needed:
```suggestion
module load cce/13.0.1
```